### ISSUE TITLE
Profile tool 4.2.0 crash

### ISSUE THREAD

**Issue by JaviBecerra** [PANOimagen/profiletool#62]
Error message:
```
Couldn't load plugin 'profiletool' due to an error when calling its classFactory() method

ImportError: cannot import name 'sip' from 'PyQt5' (C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\PyQt5\__init__.py)

Traceback (most recent call last):
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 334, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\__init__.py", line 28, in classFactory
    from .profileplugin import ProfilePlugin
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\profileplugin.py", line 41, in 
    from .tools.profiletool_core import ProfileToolCore
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\profiletool_core.py", line 48, in 
    from ..ui.ptdockwidget import PTDockWidget
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\ui\ptdockwidget.py", line 40, in 
    from ..tools.plottingtool import *
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\tools\plottingtool.py", line 44, in 
    from .. import pyqtgraph as pg
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\pyqtgraph\__init__.py", line 13, in 
    from .Qt import QtGui, mkQApp
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ee_plugin\__init__.py", line 32, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python\qgis\utils.py", line 793, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/jleon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\profiletool\pyqtgraph\Qt.py", line 137, in 
    from PyQt5 import QtGui, QtCore, QtWidgets, sip, uic
ImportError: cannot import name 'sip' from 'PyQt5' (C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\PyQt5\__init__.py)


Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
QGIS version: 3.16.10-Hannover Hannover, e5b665e952 
```
This may be due to PyQtGraph importing directly PyQt5 libraries in PyQtGraph instead of importing via qgis.PyQt, which may lead to inconsistent PyQt versions being used.

**Comment by nicogodet** [comment]
Oh I've done it all my tests on .msi/osgeo4w v2 which ship qt5.15

Could you remember me the version in your Qgis ?

**Comment by erwanC** [comment]
Hey. Same here.

QGIS | 3.16.4-Hannover
Qt | 5.11.2

**Comment by nicogodet** [comment]
Upgrading to latest PyQtGraph version should fix this (0.12.2)
Cannot test right now

**Comment by nicogodet** [comment]
Related issue https://github.com/pyqtgraph/pyqtgraph/issues/1882
